### DISCUSSION TITLE
[Feature] Add possibility to search for face name in mobile app

### DISCUSSION THREAD

**Discussion by emadivita** [immich-app/immich#11749] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In my photo archive I have a lot of faces catalogued by the person's name (using face recognition) and I would prefer to be able to search for a person in a simpler way than it is now. At the moment, from mobile, I have to scroll through all the faces (which are not even sorted alphabetically). I would like to ask if it is possible to add a search function among the faces

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by trop1kal** [comment]
Also looking for this feature. 
500+ faces tagged in my collection.

Using the mobile app and having to scroll through pictures I can never find anyone.. :(

**Reply by AlinStefan30** [reply]
Same here. It would be such a great improvement this one!

**Comment by AlinStefan30** [comment]
Either in the People section or through the Smart search would be soo good to have this one! The only thing I really missed so far.

**Comment by emadivita** [comment]
Do you have any news on this? @alextran1502 do you think this can be done in future?

**Comment by seekified** [comment]
I need this on both mobile and web - I fully expected being able to type a person's name into the main searchbar and have them show up as a suggestion dropdown or something. Having to use search filters or navigate through the People tab is tedious.

**Comment by bo0tzz** [comment]
This is now supported.
